<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogLocate</class>
 <widget class="QDialog" name="QWDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>列号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxColNo"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxColAdd">
           <property name="text">
            <string>列增</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>行号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxRowNo"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRowAdd">
           <property name="text">
            <string>行增</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>设定文字</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSetText">
        <property name="text">
         <string>设定文字</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/506.bmp</normaloff>:/images/images/506.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
